addons/sale/static/tests/tours/mail_attachment_removal_test_tour.js

```diff
@@ -0,0 +1,32 @@
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("mail_attachment_removal_tour", {
+    steps: () => [
+
+    {
+        content: "click on send by email",
+        trigger: ".o_statusbar_buttons > button[name='action_quotation_send']",
+        run: "click"
+    },
+    {
+        content: "save a new layout",
+        trigger: ".o_technical_modal button[name='document_layout_save']",
+        run: "click"
+    },
+    {
+        content: "delete attachment",
+        trigger: ".o_field_widget[name='attachment_ids'] li > button .fa-times",
```

Comment:
The `li >` part can be removed. I would even say we can remove `button`, as the `.fa-times` is more than enough.

Would be nice to not adapt this tour each time the change changes but we can still delete attachment from UI using the same icon `.fa-times`. Also shorter selector is easier to digest!